src/Payments.Core/Helpers/DbInitializer.cs

```diff
@@ -43,41 +47,10 @@ public async Task Initialize()
             var editorRole = await FindOrCreateRole(TeamRole.Codes.Editor);
             var reportRole = await FindOrCreateRole(TeamRole.Codes.ReportUser);
 
-            // create system users
-            if (!_context.Users.Any())
+            // create system user
+            if (!_context.Users.Any() && !string.IsNullOrWhiteSpace(_superuserSettings.CampusKerberos))

```

Comment:
@srkirkland @sprucely The logic hasn't changed, but with this code it never gets called in there are any users.
Maybe not how we want it to function.

Reply:
Seems like a good backstop check to me.  Really I think this code's purpose was to get a new SQL express instances started up for dev/test, so only running when the db is empty seems like a smart safe choice.